packages/amqp/test/consumers/AmqpPermissionConsumer.spec.ts

```diff
@@ -15,6 +15,52 @@ import { registerDependencies, SINGLETON_CONFIG } from '../utils/testContext'
 import { AmqpPermissionConsumer } from './AmqpPermissionConsumer'
 
 describe('AmqpPermissionConsumer', () => {
+  describe('init', () => {
```

Comment:
Adding some tests to make sure unsupported options are throwing an error and not silently accepted